libs/core/langchain_core/tracers/schemas.py

```diff
@@ -142,9 +143,14 @@ def assign_name(cls, values: dict) -> dict:
         return values
 
 
-ChainRun.update_forward_refs()
-ToolRun.update_forward_refs()
-Run.update_forward_refs()
+# TODO: Update once langsmith moves to Pydantic V2 and we can swap Run.model_rebuild
+# for Run.update_forward_refs
+with warnings.catch_warnings():
```

Comment:
I think you can undo these changes -- since it's using v1, but doesn't matter gith now. We'll likely need to delete schemas.py from the code base prior to full 0.3 release since the actual code also pulls in LLMResult (see on master)